xbee_block.py

```diff
@@ -0,0 +1,53 @@
+import serial
+import xbee
+from nio.common.block.base import Block
+from nio.common.signal.base import Signal
+from nio.common.discovery import Discoverable, DiscoverableType
+from nio.metadata.properties import StringProperty
+from nio.metadata.properties.version import VersionProperty
+from nio.modules.threading import spawn
+
+
+@Discoverable(DiscoverableType.block)
+class XBee(Block):
+
+    """ Read XBee over serial.
+
+    Parameters:
+        serial_port (str): COM/Serial port the XBee is connected to
+    """
+
+    version = VersionProperty(version='0.1.0')
+    serial_port = StringProperty(name='COM/Serial Port',
+                                 default='/dev/ttyAMA0')
+
+    BAUDRATE = 9600
```

Comment:
Hidden property sounds good. 9600 is the default for XBee but it is configurable on them.